README.md

```diff
@@ -49,6 +49,8 @@ Usage of ./endlessh-go
         Connection type. Possible values are tcp, tcp4, tcp6 (default "tcp")
   -enable_prometheus
         Enable prometheus
+  -enable_abuseipdb
```

Comment:
Is this generated by ./endlessh --helpall?

The options should be sorted alphabetically.

Reply:
I added that to the readme manually but have now used the help flag to print it out